pkg/install/deploystorage.go

```diff
@@ -263,6 +232,81 @@ func (i *Installer) deployStorageTemplate(ctx context.Context, installConfig *in
 	return i.saveGraph(ctx, g)
 }
 
+func (i *Installer) denyAssignments(clusterSPObjectID string) *arm.Resource {
+	notActions := []string{
+		"Microsoft.Network/networkSecurityGroups/join/action",
+	}
+
+	if feature.IsRegisteredForFeature(i.subscriptionDoc.Subscription.Properties, "Microsoft.RedHatOpenShift/EnableSnapshots") {
```

Comment:
Do we need to do anything to enable this `Microsoft.RedHatOpenShift/EnableSnapshots` separately?

Reply:
@mjudeikis It appears you just need to [register the feature for a subscription](https://docs.microsoft.com/en-us/rest/api/resources/features/register). @jim-minter Anything more to it than that?